src/Faker/Provider/bn_BD/Person.php

```diff
@@ -19,18 +19,18 @@ class Person extends \Faker\Provider\Person
     );
 
     protected static $firstNameMale = array(
-        'অনন্ত', 'আব্দুল্লাহ', 'আহসান',  'ইমরুল', 'করিম', 'জলিল', 'বরকত', 'মাসনুন', 'রহিম',  'রিফাত', 'হাসনাত', 'হাসান',
+        'অনন্ত', 'আব্দুল্লাহ', 'আহসান','আনাম',  'ইমরুল', 'করিম', 'জলিল', 'বরকত', 'মাহাবুব','মাসনুন', 'রহিম',  'রিফাত', 'হাসনাত', 'হাসান', 'প্রিয়' , 'পরিতোষ' , 'শুভ'
```

Comment:
This list is too short. Please add much more data, with a data source in the phpDoc comment. Same for femalefirst names and last name.